user-interface/0003-WTXN-submit_vega_transaction.md

```diff
@@ -0,0 +1,47 @@
+# Submit Vega transaction
+
+A dapp sends a transaction to a wallet, that wallet then broadcasts the transaction to a network.  Therefore the following is broken up into two steps. The transaction could fail at either. Generally: Once the transaction has gone to the network a use can use block explorer to track the transaction, but some tracking in Dapp or wallet will help.
+
+When submitting a Vega transaction of any kind, I...
+
+## Track transaction to wallet
+
+if not connected to a Vega wallet:
+
+- **must** be told that I am not connected and given the [option to connect](0012-WCON-connect_vega_wallet.md) <a name="0003-WTXN-001" href="#0003-WTXN-001">0003-WTXN-001</a>
+
+if transaction not auto approved by wallet:
+
+- **must** see a prompt to check connected vega wallet to approve transaction <a name="0003-WTXN-002" href="#0003-WTXN-002">0003-WTXN-002</a>
+
+if order is approved by wallet:
+
+- **must** see A hash/transaction ID <a name="0003-WTXN-003" href="#0003-WTXN-003">0003-WTXN-003</a>
```

Comment:
as well as a label telling you what it is, with naming that is consistent across dapps

Reply:
explanation added in https://github.com/vegaprotocol/specs/pull/1181/commits/8fa3efed0a99c1bf2ac27c3eeb9775b70ffe1760
```suggestion
- **must** see A [transaction hash](7001-DATA-data_display.md#transaction-hash) <a name="0003-WTXN-003" href="#0003-WTXN-003">0003-WTXN-003</a>
```